### ISSUE TITLE
[FEATURE] CustomHUD ChatFont

### ISSUE THREAD

**Issue by DeleteAndInsert** [CCBlueX/LiquidBounce1.8-Issues#3008]
This can be changed in the HUD, but what about your own font?

**Comment by mrphax0r** [comment]
Just replace the .ttf file

**Comment by mrphax0r** [comment]
I think

**Comment by RealFantaCoke** [comment]
> Just replace the .ttf file

No, because they are getting downloaded again from the [LiquidBounce FileCloud](https://github.com/CCBlueX/FileCloud/tree/master/LiquidBounce)

**Comment by SenkJu** [comment]
You can replace it. There is no integrity check for the font files.
